blog/_posts/2019-06-27-commit-of-the-month-june.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: june-tilt-commit-of-the-month
+date: 2019-07-03T15:26:34.743Z
+author: dmiller
+layout: blog
+title: "Tilt Commit of the Month"
```

Comment:
consider a title (or subtitle) with a bit more context, e.g. "Commit of the Month: Support More k8s Clusters"